src/containers/EnterpriseApp/EnterpriseApp.test.jsx

```diff
@@ -123,6 +125,7 @@ describe('EnterpriseApp', () => {
 
     // ensure focus is set on content wrapper
     expect(document.activeElement.className).toEqual('content-wrapper');
+    global.innerWidth = initialWidth;
```

Comment:
don't have context on this setting here, but in case this line does not execute and somehow affects other tests, is it better in a teardown or after method?

Reply:
I didn't write this test; I just added this because it didn't looks like this was ever reset. I'm going to call it better than it used to be. ;)

Reply:
Totally! my point was mainly about the resetting of the innerWidth would ideally be in cleanup, but since you ran the test and it passed without side effects, no worries.